plugins/rating/test/rating.spec.js

```diff
@@ -125,7 +129,7 @@ test.serial('gets aggregated rating stats with ranking', async (t) => {
       t.is(typeof result.ranking, 'number')
       t.is(typeof result.lowestRanking, 'number')
 
-      t.is(result.lowestRanking, obj.nbResults) // is true because there is no filter
+      t.is(result.lowestRanking, obj.results.length) // is true because there is no filter
```

Comment:
Don’t forget than removing `nbResults` is a also one of the breaking changes.

Did we make this choice to avoid redundancy and even confusion with total number of results?

Reply:
This `nbResults` property from pagination meta represents the total number of results (which is different from `nbResultsPerPage`) that won't be available with cursor pagination.

Reply:
Yes, it was confusing as it could also be mistakenly seen as the number of results of the current page (can be less than `nbResultsPerPage`).

Reply:
Yes it is. This test rely on the relative few number of aggregated results (less than 20 results).
I'll add a comment for it.

Reply:
No comment needed.

The confusing part I’m referring to is the deprecated `nbResults`.